### ISSUE TITLE
Clicking the "Add-ons for Android" from "More" menu will not refresh the homepage 

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-frontend#6650]
STR:
1. Load AMO dev and observe the "Featured extensions" shelf from the homepage
2. Open the "More" Menu and click on "Add-ons for Android" 
3. Refresh page and observe again the "Featured extensions" shelf
4. Repeat this with "Add-ons for Firefox" 

Expected result:
The page is automatically refreshed after clicking the "Add-ons for Android" from More. 

Actual result:
The featured extension shelf still displays extensions. Only after refreshing the page these will no longer be displayed.  When  "Add-ons for Firefox" is clicked the page should display the available extensions from featured extensions shelf automatically. 

Notes:
This issue is reproducible on AMO dev and AMO stage with FF62 - Win10&Android 8.0

![updated only after refresh](https://user-images.githubusercontent.com/33448286/47021855-f577bd00-d164-11e8-94e6-d1e7b3ce43a1.gif)

**Comment by AlexandraMoga** [comment]
This looks like a duplicate of https://github.com/mozilla/addons-frontend/issues/3453  - slightly different STR but the effect is the same

**Comment by ioanarusiczki** [comment]
This scenario is no longer reproducible on AMO dev and AMO stage with FF64(Win10). 

![featured extensions work](https://user-images.githubusercontent.com/33448286/50906618-27c84c00-142e-11e9-85ee-fb1e04f7e6bf.gif)
